test/Datadog.Trace.ClrProfiler.IntegrationTests/AspNetMvc4Tests.cs

```diff
@@ -1,73 +1,42 @@
-using System;
-using System.Collections.Generic;
-using System.Diagnostics;
-using System.IO;
-using System.Linq;
+#if NET461
+
 using System.Net;
-using System.Threading;
 using System.Threading.Tasks;
-using Datadog.Trace.TestHelpers;
 using Xunit;
 using Xunit.Abstractions;
 
 namespace Datadog.Trace.ClrProfiler.IntegrationTests
 {
-    public class AspNetMvc4Tests : TestHelper
+    [Collection("IIS test collection")]
+    public class AspNetMvc4Tests : TestHelper, IClassFixture<IisFixture>
     {
-        private const int AgentPort = 9006;
-        private const int Port = 9007;
+        private readonly IisFixture _iisFixture;
 
-        public AspNetMvc4Tests(ITestOutputHelper output)
+        public AspNetMvc4Tests(IisFixture iisFixture, ITestOutputHelper output)
             : base("AspNetMvc4", output)
         {
+            _iisFixture = iisFixture;
+            _iisFixture.StartIis(this);
         }
 
-        [Fact]
+        [Theory]
         [Trait("Category", "EndToEnd")]
-        public void SubmitsTraces()
+        [Trait("Integration", nameof(Integrations.AspNetMvcIntegration))]
+        [InlineData("/Home/Index", "GET home.index")]
```

Comment:
Would it make sense to test
```
[InlineData("/delay/0", "GET home.delay")]
[InlineData("/delay-async/0", "GET home.delayasync")]
```
for AspNetMvc4 too?

Reply:
Yeah. In fact, I'm going to throw away the current MVC4 sample app and build a new one that's a copy of the MVC5 sample.

Reply:
... in a separate PR.